caffeine/src/main/scala/scalacache/caffeine/Entry.scala

```diff
@@ -0,0 +1,15 @@
+package scalacache.caffeine
+
+import org.joda.time.DateTime
+
+/**
+ * A cache entry with an optional expiry time
+ */
+case class Entry[+A](value: A, expiresAt: Option[DateTime]) {
```

Comment:
Let's move this class to the core project and share it across Guava and Caffeine